src/main/java/com/xceptance/neodymium/module/statement/browser/multibrowser/configuration/BrowserConfiguration.java

```diff
@@ -27,6 +29,12 @@ public class BrowserConfiguration
 
     private List<String> arguments;
 
+    private Map<String, Boolean> preferencesBoolean;
```

Comment:
orginaly it was because firefox doesn't offer a general method to set preferences and requires parsing to one of the types (boolean, int or string) but the parsing can be then done directly by setting preferences. Adjusted the code to do so.